### ISSUE TITLE
Wiki Checker: use Github API in order to avoid rate limiting

### ISSUE THREAD

**Issue by Alexandru-Dascalu** [ISISComputingGroup/IBEX#5363]
As a developer, I would like that our Wiki Checker tests do not fail when checking if github links are valid because Github limits calls to 300 an hour. 

~We should get a Github API key and change the wiki checker code so it uses the Github API for checking the web pages, which will go around the rate limiting issue.~ - This is not possible with the Github API, to workaround we should whitelist the fully-fledged URLs which link to the wikis and dynamically check against the cloned copies of the wikis instead. 

## Acceptance Criteria
1. Github links are checked by the wiki checker
2. Full links to our own wiki pages are handled dynamically and are checked against the local, checked-out wikis instead of calling requests to Github.

**Comment by Alexandru-Dascalu** [comment]
A a result of https://github.com/ISISComputingGroup/ibex_wiki_checker/pull/8, all github links are white listed and are not being checked. But this is a temporary fix that should be rolled back once this ticket is merged into master.
